sweepai/handlers/on_ticket.py

```diff
@@ -140,6 +145,28 @@ def strip_sweep(text: str):
         re.search(r"^[Ss]weep\s?\([Ff]ast\)", text) is not None,
     )
 
+workers = []
+worker_stop_signals = []
```

Comment:
Unfortunately the memory is not shared across Sweep runs. Please utilize Modal's shared dictionary:

https://github.com/sweepai/sweep/blob/2efeaad8e6b81f62932b4c06a820d9b3b3cb2a48/sweepai/api.py#L104

https://modal.com/docs/reference/modal.Dict